pandas/io/tests/test_network.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+
+"""
+Tests parsers ability to read and parse non-local files
```

Comment:
Ah, okay, I'll move all of those (excluding `test_lib` of course and the duplicate `test_parsers`) into the `parser` directory as well